ui/StatusQ/src/StatusQ/Core/Utils/AmountsArithmetic.qml

```diff
@@ -86,12 +86,12 @@ QtObject {
 
       The obtained amount can be multiplied or compared.
 
-      Provided number is assumed to be an amount in basic units, an integer.
+      Provided number is assumed to be an amount in basic units, e.g. an integer or a float number.
 
       returns NaN in case the conversion fails.
      */
     function fromString(numStr) {
-        console.assert(typeof numStr === "string")
+        console.assert(typeof numStr === "string" || typeof numStr === "number")
```

Comment:
this function clearly says fromString, I dont think we should do this here. @micieslak thoughts?

Reply:
That CTOR accepts more than a string, there's no reason to `assert` here :) See https://mikemcl.github.io/big.js/#big

Reply:
The naming was wrong from the start, we just didn't notice

Reply:
but its the first time we dont pass a string but a number to this function so we are extending the function?

Reply:
No, not really extending but correcting the expectations I'd say... not sure, I can remove it if you insist :)